USE_CASES.md

```diff
@@ -171,6 +172,186 @@ print(response.body)
+    response = sg.client.mail.send.post(request_body=mail.get())
+
+    return HttpResponse('Email Sent!')
+```
+
+Now the folder structure should look like this:
+
+```
+hello-sendgrid
+├── hello_sendgrid
+│   ├── hello_sendgrid
+│   │   ├── __init__.py
+│   │   ├── settings.py
+│   │   ├── urls.py
+│   │   ├── views.py
+│   │   └── wsgi.py
+│   └── manage.py
+└── requirements.txt
+```
+
+Next we open the file `urls.py` in order to add the view we have just created to the Django URL dispatcher.
+
+```python
+from django.conf.urls import url
+from django.contrib import admin
+
+from .views import index
+
+
+urlpatterns = [
+    url(r'^admin/', admin.site.urls),
+    url(r'^sendgrid/', index, name='sendgrid'),
+]
+```
+
+We also assume that you have set up your development environment with your `SENDGRID_API_KEY`. If you have not done it yet, please do so. See the section [Setup Environment Variables](https://github.com/sendgrid/sendgrid-python#setup-environment-variables).
+
+Now we should be able to send an email. Let's run our Django development server to test it. Find the file `manage.py` then run:
+
+```
+$ python manage.py runserver
+```
+
+By default, it starts the development server at `http://127.0.0.1:8000/`. To test if we can send email or not, go to `http://127.0.0.1:8000/sendgrid/`. If it works, we should see the page says "Email Sent!".
+
+## Deploy to Heroku
+
+Before we start the deployment, let's log in to your Heroku account and create a Heroku app. This tutorial uses `hello-sendgrid`.
+
+We also need to do a couple things:
+
+1. Add `'*'` or your Heroku app domain to `ALLOWED_HOSTS` in the file `settings.py`.
+2. Add `Procfile` with the code below to declare what commands are run by your application's dynos on the Heroku platform.
+
+```
+web: cd hello_sendgrid && gunicorn hello_sendgrid.wsgi --log-file -
+```
+
+The final folder structure looks like this:
+
+```
+hello-sendgrid
+├── hello_sendgrid
+│   ├── hello_sendgrid
+│   │   ├── __init__.py
+│   │   ├── settings.py
+│   │   ├── urls.py
+│   │   ├── views.py
+│   │   └── wsgi.py
+│   └── manage.py
+├── Procfile
+└── requirements.txt
+```
+
+There are different deployment methods we can choose. In this tutorial, we choose to deploy our app using the [Heroku CLI](https://devcenter.heroku.com/articles/heroku-cli). Therefore, let's install it before we go further.
+
+Once you have the Heroku CLI installed, run the command below to log in to your Heroku account if you haven't already.
+
+```
+$ heroku login
+```
+
+Go to the root folder then initialize a Git repository.
+
+```
+$ git init
+$ heroku git:remote -a hello-sendgrid
+```
+
+Since we do not use any static files, we will disable `collectstatic` for this project.
+
+```
+$ heroku config:set DISABLE_COLLECTSTATIC=1
+```
+
+Commit the code to the repository and deploy it to Heroku using Git.
+
+```
+$ git add .
+$ git commit -am "Create simple Hello Email Django app using SendGrid"
+$ git push heroku master
```

Comment:
I got to here and then got this error, I tried creating a runtime.txt file and that didn't change anything. 

```
remote: Compressing source files... done.
remote: Building source:
remote:
remote:  !     No default language could be detected for this app.
remote:                         HINT: This occurs when Heroku cannot detect the buildpack to use for this application automatically.
remote:                         See https://devcenter.heroku.com/articles/buildpacks
remote:
remote:  !     Push failed
remote: Verifying deploy...
remote:
remote: !       Push rejected to hello-sendgrid-django-python.
remote:
To https://git.heroku.com/hello-sendgrid-django-python.git
 ! [remote rejected] master -> master (pre-receive hook declined)
error: failed to push some refs to 'https://git.heroku.com/hello-sendgrid-django-python.git'
```

Reply:
I found a bug in my tutorial. Previously, I said:

```bash
$ pip freeze > requirements.text
```

The issue was Heroku couldn't detect `requirements.text`. It actually should be `requirements.txt`. I have fixed that already.